webcompat/webhooks/helpers.py

```diff
@@ -230,3 +234,35 @@ def prepare_rejected_issue():
     payload_request['state'] = 'closed'
     payload_request['milestone'] = invalid_id
     return payload_request
+
+
+def make_classification_request(issue_number):
+    url = f"{BUGBUG_HTTP_SERVER}/{CLASSIFIER_PATH}/{issue_number}"
+    headers = {"X-Api-Key": "webcompat"}
+    response = requests.get(url, headers=headers)
+    response.raise_for_status()
+    return response
+
+
+def get_issue_classification(issue_number, retry_count=4, retry_sleep=3):
+    """Get issue classification from bugbug.
+
+    As classification happens in the background we need to make a second
+    request to get results.
+
+    The service returns 202 status if request is still in process
+    and 200 status if the issue is classified
+    """
+    for _ in range(retry_count):
+        response = make_classification_request(issue_number)
+
+        if response.status_code == 202:
+            time.sleep(retry_sleep)
+        else:
+            break
```

Comment:
`202` code in this code is a wait instruction. aka the response is not ready. 
`200` and `4**` and `500` will be treated the same. 

Is it what we want?

Reply:
Yeah, this code will basically keep retrying if the status is 202 up to a certain limit. If it's 200, 4** or 500, we don't need to retry. And in case of 4** or 500 the error will be raised in `make_classification_request`